<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>321</height>
     </rect>
    </property>
    <property name="title">
     <string>设备列表</string>
    </property>
    <widget class="QListWidget" name="lw_device">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>141</width>
       <height>301</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>551</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>数据接口</string>
    </property>
    <widget class="QLineEdit" name="le_url">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_get">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>获取</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>接口URL：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_name">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_mac">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>60</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_name">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>设备名称</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_mac">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>64</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>设备地址</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>551</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>消息列表</string>
    </property>
    <widget class="QTextBrowser" name="tb_msg">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>12</y>
       <width>551</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>722</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="action_URL"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_URL">
   <property name="text">
    <string>修改URL</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>清空消息列表</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>清空设备列表</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>清空设备信息</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
